docs/guides/haiku_migration_guide.rst

```diff
@@ -861,4 +861,47 @@ as we are using ``5`` layers.
                   kernel: (5, 64, 64),
               },
           },
-      })
\ No newline at end of file
+      })
+
+Raw Haiku functions vs Flax modules
+----------------------
+
+In Haiku, `hk.Module` is not required, and writing a forward function to be passed into `hk.transform` is enough. In this process, people use raw `hk.<get|set>_<parameter|state>` functions to access model data and states.
```

Comment:
Here is a suggestion for the wording 😅:
```suggestion
In Haiku, it is possible to write the entire model as a single function by using ``hk.{get,set}_{parameter,state}`` to define/access model parameters and states. It very common to write the top-level "Module" as a function instead:
```
If you prefer the current wording, you do need to fix the "inline code" syntax which in .rst is using double back tick: "\`\`"